packages/zoe/src/contracts/callSpread/fundedCallSpread.js

```diff
@@ -0,0 +1,136 @@
+// @ts-check
+import '../../../exported';
+import './types';
+
+import { assert, details } from '@agoric/assert';
+import { makePromiseKit } from '@agoric/promise-kit';
+import { E } from '@agoric/eventual-send';
+import {
+  assertProposalShape,
+  depositToSeat,
+  trade,
+  assertUsesNatMath,
+} from '../../contractSupport';
+import { makePayoffHandler } from './payoffHandler';
+import { Position } from './position';
+
+/**
+ * This contract implements a fully collateralized call spread. This is a
+ * combination of a call option bought at one strike price and a second call
+ * option sold at a higher price. The invitations are produced in pairs, and the
+ * purchaser pays the entire amount that will be paid out. The individual
```

Comment:
```suggestion
 * instance creator escrows upfront all collateral that will be paid out. The individual
```

We shouldn't be talking about "purchasing" or "buy" since no buying is happening in any of the call spread contracts.

Reply:
done